packages/search/graphql/resolvers/_queries/search.js

```diff
@@ -354,12 +357,14 @@ const search = async (__, args, context, info) => {
   } = args
 
   // detect if Document.content is requested
-  let withoutContent
-  if (info) {
-    withoutContent = _withoutContent !== undefined ||
-      !hasFieldRequested('Document.content', info)
-  } else {
-    withoutContent = _withoutContent || false
+  let withoutContent = _withoutContent || false
+  let withoutChildren = withoutContent
+  if (info && _withoutContent === undefined) {
+    withoutContent = !hasFieldRequested('content', info)
```

Comment:
Wouldn't this __wired__ query trigger content even though it's not needed?
`{ documents {nodes {meta {ownDiscussion {comments {nodes {content}}}}}}}`

Reply:
Probably. Maybe `hasFieldRequested` could be extended to support a max depth but would require a unit test approach to ensure correct behaviour between `search`, `documents` and `document`. IMO ignore for now since unlikely. What do you think?